backend/apps/billing/tasks.py

```
@@ -0,0 +1,211 @@
+        Paragraph(f"<b>Date:</b> {invoice_date}", text_style),
+        Paragraph(
+            (
+                f"<b>Status:</b> <font color='green'><b>{invoice.status.upper()}</b></font>"
+                if invoice.status == "paid"
+                else f"<b>Status:</b> {invoice.status.upper()}"
+            ),
+            text_style,
+        ),
+    ]
+    metadata_right = [
+        Paragraph(f"<b>Billed To:</b>", bold_text_style),
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)